src/model/managing.officer.corporate.model.ts

```diff
@@ -13,6 +13,7 @@ export const ManagingOfficerCorporateKeys: string[] = [
   "public_register_name",
   "registration_number",
   "start_date",
+  "role_and_responsibilities",
```

Comment:
The story refers to 'roles and responsibilities' so this should be reflected in the field name here (and elsewhere), unless story is wrong...